lib/awful/placement.lua

```diff
@@ -276,6 +280,30 @@ local outer_positions = {
     bottom_middle = function(r, w, _) return {x=r.x-w/2+r.width/2, y=r.y                }, "middle" end,
 }
 
+-- Map the opposite side for a string
+local opposites = {
```

Comment:
(This is relevant to all 3 of new locals tables)

Do you think we should expose them as `awful.placement` static fields? I think it could be interesting to prevent duplication of these tables in other modules, like you did here with wibar. It would also "make sure" that the values are correct/consistent across the API.